src/main/java/com/quimify/api/health/HealthResult.java

```diff
@@ -4,18 +4,18 @@
 import java.util.List;
 
 public class HealthResult {
-    private boolean present;
+    private boolean healthy;
     private String message;
     private List<String> errors;
 
-    public HealthResult(boolean present, String message) {
```

Comment:
Use two (three) constructors:
1. Public one with a list parameter
2. Named constructor healthy()
3. Private constructor for number 2 to work